src/main/java/com/internet/shop/controller/RegistrationUserController.java

```diff
@@ -0,0 +1,42 @@
+package com.internet.shop.controller;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.model.ShoppingCart;
+import com.internet.shop.model.User;
+import com.internet.shop.service.ShoppingCartService;
+import com.internet.shop.service.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class RegistrationUserController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private UserService userService = (UserService) injector.getInstance(UserService.class);
+    private ShoppingCartService cartService = (ShoppingCartService) injector
+            .getInstance(ShoppingCartService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/user/registration.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        if (password.equalsIgnoreCase(req.getParameter("repeatPwd"))) {
```

Comment:
do we want password123 and PassWorD123 be the same passwords?

Reply:
done